themes/digital.gov/src/scss/new/_communities.scss

```diff
@@ -9,6 +9,23 @@
   }
 }
 
+.communities__home {
+  @include u-flex("row", "wrap");
+  display: flex;
+  justify-content: center;
+  padding-top: units(2);
+  &-title {
+    margin-bottom: units(-5);
+    margin-top: units(5);
+  }
+}
+
+.usa-card.home {
```

Comment:
We could improve clarity by moving home specific styles to their respective page partials, like`/scss/new/home/_home.scss`.